### DISCUSSION TITLE
Yボタンの位置の修復方法

### DISCUSSION THREAD

**Discussion by tattomo3** [Yowkees/keyball#425] [General]
お世話になっています。
今回、keyball39を購入させていただきました。
作業時に赤枠の部分の基盤の銅線が剥離してしまったため、はんだでだまを作ることで、なんとか利用できていたのですが、先日反応しなくなってしまいました。
無知なもので、どう対処すればよいかわからないため、教えていただきたいです。

![keyball39_right_withball](https://github.com/Yowkees/keyball/assets/128353854/8738afe4-0fa9-4b79-9d8f-60c58197a0b3)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

状況、了解しました。
下記画像の緑点どうしをエナメル線やゴム被覆導線で接続していただければ復活できます。
なお、キースイッチを差し込むときにキーソケットが脱落しないようにテープ等で固定した方が良いかもしれません。

以上、お手数ですがご検討よろしくお願いいたします。

![KB39_Yfix](https://github.com/Yowkees/keyball/assets/84209772/8bc26d07-6b24-4562-bcc2-b0cecb1ea640)
